src/Bravo3/OrmBundle/Services/Factories/OrmFactory.php

```diff
@@ -15,11 +15,12 @@ class OrmFactory
      * @param string          $cache_dir
      * @return EntityManager
      */
-    public static function createEntityManager(DriverInterface $driver, $cache_dir)
+    public static function createEntityManager(DriverInterface $driver, $cache_dir, $hydration_exceptions_as_events)
```

Comment:
Give `$hydration_exceptions_as_events` it a default value of `false` to prevent BC breaks